script/release

```diff
@@ -96,4 +86,7 @@ x = STDIN.gets
 exit(1) unless x.strip == 'yes'
 puts
 
-verbose("git push origin HEAD:release --follow-tags -f")
+verbose("git push origin HEAD:release -f")
+
+Dir.chdir(File.expand_path(File.join(File.dirname(__FILE__), '../demos/serverless')))
+verbose("npm run deploy -- -b chime-sdk-demo-release-testing -s chime-sdk-demo-release-testing")
```

Comment:
Deploy to a stack for the relevant version number.

Reply:
Updated